### PR TITLE
media-libs/libbluray: 1.0.2 version bump

### PR DESCRIPTION
**Pull request by candrews**
https://bugs.gentoo.org/640406

### CODE REVIEW

File: media-libs/libbluray/libbluray-1.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		$(use_with truetype freetype) \
+		$(use_enable static-libs static) \
+		$(use_with xml libxml2)
+}
+
+multilib_src_install() {
+	emake DESTDIR="${D}" install
+
+	if multilib_is_native_abi && use utils; then
+		cd .libs/
```
Comment (Moiman) [inline media-libs/libbluray/libbluray-1.0.2.ebuild]: Missing die

File: media-libs/libbluray/libbluray-1.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	fi
+
+	if multilib_is_native_abi && use java; then
+		java-pkg_dojar "${BUILD_DIR}"/.libs/${PN}-j2se-*.jar
+	fi
+}
+
+multilib_src_install_all() {
+	einstalldocs
+	prune_libtool_files
```
Comment (Moiman) [inline media-libs/libbluray/libbluray-1.0.2.ebuild]: find "${D}" -name '*.la' -delete || die
and eutils aren't probably needed after this.

File: media-libs/libbluray/libbluray-1.0.2.ebuild (hunk 1 above)
Comment (candrews) [reply media-libs/libbluray/libbluray-1.0.2.ebuild]: good catch, thanks!

File: media-libs/libbluray/libbluray-1.0.2.ebuild (hunk 2 above)
Comment (candrews) [reply media-libs/libbluray/libbluray-1.0.2.ebuild]: 👍

**Comment by candrews** [comment]
@LordVan So are you okay with this PR now?

I'm planning on merging this PR on December 16 unless I hear otherwise.

**Comment by candrews** [comment]
https://github.com/gentoo/gentoo/commit/007154e9f09da768ebc42a4ad0e88a650e3bad3f

**Pull request closed without merging**